sample.env

```diff
@@ -30,13 +22,33 @@ DOCKER_HOST_IP=
 # (Required) The full path of a directory on the host machine to be shared with docker containers.
 # Hint: usually the path to the `jpo-ode` directory.
 DOCKER_SHARED_VOLUME=
-DOCKER_SHARED_VOLUME_WINDOWS=
+
+# Docker compose restart policy: https://docs.docker.com/engine/containers/start-containers-automatically/
+RESTART_POLICY="on-failure:3"
 
 # (Required if values are not sent in REST request JSON messages)
 # RSU SNMP username and password
 ODE_RSU_USERNAME=
 ODE_RSU_PASSWORD=
 
+# (Required values to define what services spin up)
+# Available profiles:
+# - all
+#   - ode_base
+#       - ode
+#       - adm
+#       - aem
+#   - ode_full
+#       - ode
+#       - adm
+#       - aem
+#       - ppm_bsm
+#       - sdw_depositor
+#       - sec
+# All profiles from the jpo-utils repo are also allowed
+# EXAMPLE: COMPOSE_PROFILES=ode_base,kafka_connect_standalone,kafka_setup
+COMPOSE_PROFILES=ode_base,kafka,kafka_setup
```

Comment:
Where do these kafka and kafka_setup profiles get used?

Reply:
That's outlined in the jpo-utils [readme](https://github.com/usdot-jpo-ode/jpo-utils/tree/develop)/[sample.env](https://github.com/usdot-jpo-ode/jpo-utils/blob/develop/sample.env) file. The idea behind this is to reduce the amount of information in a single `.env` file and instead spread it across multiple. The include part of the docker-compose will include the child `.env` files in the context.

Reply:
Can we add some documentation around this here as well then? It's confusing to see these listed here but not be able to find in code/yaml where it's being implemented.

Reply:
The though that I had was that we want to limit duplication of documentation to reduce the areas that had to be updated when changes are made. I have a bit of documentation on the `COMPOSE_PROFILE` env variable in both the jpo-ode readme (lines 327-337) as well as in the jpo-utils readme (mongo: 58-60, kafka: 82-86, kafka-connect 125-130).

The implementation of the `COMPOSE_PROFILE` can be seen in the actual `docker-compose.yml` files. Each service has a `profiles` field that includes a list of profiles that it will deploy with.

Reply:
Ah, I hadn't passed in the "--recurse-submodules" with my first pull so I wasn't see _any_ of that documentation. Maybe just a simple note in here to direct the user to the proper documentation? We've got it mentioned that we can pull in the profiles from utils, but maybe just "See utils README for documentation of these profiles"